### PR TITLE
feat: Add override_columns to openapi transformer

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: cloudquery/sdk/transformers/openapi.py [hunk 1]
```diff
@@ -24,11 +24,24 @@ def oapi_type_to_arrow_type(field) -> pa.DataType:
...
     columns = []
     for key, value in definition["properties"].items():
         column_type = oapi_type_to_arrow_type(value)
-        columns.append(
-            Column(name=key, type=column_type, description=value.get("description"))
+        column = Column(
+            name=key, type=column_type, description=value.get("description")
         )
+        override_column = get_column_by_name(override_columns, key)
+        if override_column is not None:
```
Comment (disq) [inline cloudquery/sdk/transformers/openapi.py:42]:
```suggestion
        if override_column:
```

File: cloudquery/sdk/transformers/openapi.py [hunk 2]
```diff
@@ -24,11 +24,24 @@ def oapi_type_to_arrow_type(field) -> pa.DataType:
         return pa.string()
 
 
-def oapi_definition_to_columns(definition: Dict) -> List[Column]:
+def get_column_by_name(columns: List[Column], name: str) -> Column:
```
Comment (hermanschaaf) [inline cloudquery/sdk/transformers/openapi.py:27]:
```suggestion
def get_column_by_name(columns: List[Column], name: str) -> Optional[Column]:
```

(will require `from typing import Optional`)

**Review by hermanschaaf — APPROVED** [review]

File: cloudquery/sdk/transformers/openapi.py [hunk 3]
```diff
@@ -24,11 +24,24 @@ def oapi_type_to_arrow_type(field) -> pa.DataType:
...
+def oapi_definition_to_columns(definition: Dict, override_columns=[]) -> List[Column]:
     columns = []
     for key, value in definition["properties"].items():
         column_type = oapi_type_to_arrow_type(value)
-        columns.append(
-            Column(name=key, type=column_type, description=value.get("description"))
+        column = Column(
+            name=key, type=column_type, description=value.get("description")
         )
+        override_column = get_column_by_name(override_columns, key)
```
Comment (hermanschaaf) [inline cloudquery/sdk/transformers/openapi.py:41]: Probably `override_columns.find(key)` or something like that should also work here

File: cloudquery/sdk/transformers/openapi.py (hunk 1 above)
Comment (yevgenypats) [reply cloudquery/sdk/transformers/openapi.py:42]: I think better `is not None` in python in case it evaluates to false

Comment (disq) [reply cloudquery/sdk/transformers/openapi.py:42]: false would be falsy, also it's either a `Column` or `None`? I think this is the Pythonic way.

**Pull request merged**